crates/file_finder/src/file_finder.rs

```diff
@@ -204,15 +204,35 @@ impl PartialOrd for ProjectPanelOrdMatch {
 #[derive(Debug, Default)]
 struct Matches {
     separate_history: bool,
+    opened_path: Option<FoundPath>,
```

Comment:
I wonder if we really need to store this path?
It seems to be the same as `currently_opened_path` which could be used directly after this change, as we pass it as a parameter.